src/main/config/config.c

```diff
@@ -454,6 +454,8 @@ static void resetConf(void)
     masterConfig.gyro_sync_denom = 4;
 #endif
     masterConfig.gyro_soft_lpf_hz = 100;
```

Comment:
@borisbstyle , I think it's not worth it packing these into a struct - if we do parameter groups in v3.1 then all this will be cleaned up anyway.